okapi-core/src/main/java/org/folio/okapi/service/impl/PostgresHandle.java

```diff
@@ -56,35 +53,22 @@ protected PostgresHandle(Vertx vertx, JsonObject conf) {
         Integer x = Integer.parseInt(val);
         pgconf.put("port", x);
       } catch (NumberFormatException e) {
-        logger.warn("Bad postgres_port value: " + val + ": " + e.getMessage());
+        logger.warn("Bad postgres_port value: {}: {}", val, e.getMessage(), e);
```

Comment:
Can you remove `, e`? Adding a stacktrace for this NumberFormatException doesn't make sense.